Formula/tuir.rb

```diff
@@ -0,0 +1,23 @@
+class Tuir < Formula
+  include Language::Python::Virtualenv
+
+  desc "Command-line Reddit client"
+  homepage "https://gitlab.com/ajak/tuir"
+  url "https://gitlab.com/ajak/tuir/-/archive/master/tuir-master.tar.gz"
```

Comment:
This is not acceptable, the download resource should be stable. https://docs.brew.sh/Acceptable-Formulae#stable-versions